dist/docker-images/compose.test.bash

```diff
@@ -13,17 +13,18 @@ cleanup(){
 
 portcheck(){
     PORT="${1}"
-    if [ "$(nc -zv -G 5 localhost "$PORT" 2>&1 | grep -c 'succeeded')" -lt 1 ]; then
```

Comment:
Always returns error on Linux which is interpreted here as the port is available even if it is not available. This is because `-G` isn't valid for  the OpenBSD netcat or Nmap's ncat which are both commonly used in Linux.

Refactored to trust the exit code when doing a simple SYN check.

Reply:
nice